package/MDAnalysis/lib/transformations.py

```diff
@@ -109,6 +109,8 @@
 Examples
 --------
 
+>>> from MDAnalysis.lib.transformations import *
+>>> import numpy
```

Comment:
I think if would be better to change the whole test to be consistent with the others and use `import numpy as np`.